### ISSUE TITLE
Toggle /ad

### ISSUE THREAD

**Issue by Tringlle** [Minehut/Meta#633]
### Feature request

**Checklist**
- [ ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ] This is not a bug report, support request or plugin request
- [ ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ] This issue does not contain any sensitive information 

**Platform**

Lobbies
**Describe the feature**

Allow server owners to toggle /ad. So when it’s disabled, people can not /ad that server.
**How the feature is useful**

This can be useful for things like whitelisted smps and just general owners who don’t want /ad.
---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by not-3than** [comment]
I don’t think this should be added. Advertisements are pretty much the only reason I use the lobbies, and I’m sure it’s the same for loads of other people. Lobbies are a driving factor of what makes Minehut a better place to grow a community, and if people shut off opportunities to join communities, this is bad for Minehut in terms of keeping players on the network, and bad for player servers in terms of getting players.

**Comment by quick007** [comment]
:clown:

**Comment by ahoulton** [comment]
The admins/developers have already said that they will not add a "toggle advertisements" system, you may use client sided mods to block the messages however if you really wanted to.

**Comment by not-3than** [comment]
> The admins/developers have already said that they will not add a "toggle advertisements" system, you may use client sided mods to block the messages however if you really wanted to.

you didn’t read what they wrote, did you

**Comment by Tringlle** [comment]
> The admins/developers have already said that they will not add a "toggle advertisements" system, you may use client sided mods to block the messages however if you really wanted to.

This isnt what I meant. Read the post all the way this time.


Summary: Make a control in the console to disallow /ad for the specified server. People can not /ad <servername> if <servername> has this setting set to "Off"

**Comment by iGalaxyYT** [comment]
At the very least, only allow the linked Minecraft account to use /ad for a given server (toggleable with a setting of course)
